src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -1891,14 +1922,13 @@ public EntryResult tMedicationActivity2MedicationStatement(MedicationActivity cd
 		}
 
 		// consumable.manufacturedProduct -> medication
+
 		if (cdaMedicationActivity.getConsumable() != null && !cdaMedicationActivity.getConsumable().isSetNullFlavor()) {
-			if (cdaMedicationActivity.getConsumable().getManufacturedProduct() != null
-					&& !cdaMedicationActivity.getConsumable().getManufacturedProduct().isSetNullFlavor()) {
-				Bundle fhirMedicationBundle = tManufacturedProduct2Medication(
-						cdaMedicationActivity.getConsumable().getManufacturedProduct());
-				for (BundleEntryComponent entry : fhirMedicationBundle.getEntry()) {
-					result.addResource(entry.getResource());
-					if (entry.getResource() instanceof org.hl7.fhir.dstu3.model.Medication) {
+			EntryResult fhirMedicationResult = tManufacturedProduct2Medication(cdaMedicationActivity.getConsumable());
+			if (fhirMedicationResult.hasResult()) {
+				result.updateFrom(fhirMedicationResult);
+				for (BundleEntryComponent entry : fhirMedicationResult.getBundle().getEntry()) {
+					if (entry.getResource() instanceof Medication) {
```

Comment:
can you add back in the full name of the Med instance of reference?  I feel like I made this change before.

Reply:
Not clear on this... What do you mean by the full name of the med instance?

Reply:
oh, with the package included. Got it.